src/gui/properties/propertieswidget.cpp

```diff
@@ -94,17 +92,17 @@ PropertiesWidget::PropertiesWidget(QWidget *parent, MainWindow *main_window, Tra
 
     // SIGNAL/SLOTS
     connect(m_ui->filesList, SIGNAL(clicked(const QModelIndex&)), m_ui->filesList, SLOT(edit(const QModelIndex&)));
```

Comment:
@thalieht:  Can you also upgrade all these `SIGNAL`/`SLOT` lines to the Qt5 `connect` syntax?

Reply:
Link:  [<https://wiki.qt.io/New_Signal_Slot_Syntax>]

Reply:
I'm afraid not. I've seen the other devs change a few and some are straightforward but some are "What is going on here!".

Reply:
@thalieht:  

>some are straightforward but some are "What is going on here!" 

Have you tried automatically converting them (using the built-in tool of Qt Creator)?

Reply:
At least, I think there is such a built-in tool.  I may be getting confused with something else.

Reply:
I didn't know that but still i don't think it's a good idea because i wouldn't know how to fix something that might go wrong. Anyway IIRC evsh wanted to do it after 4.0.0.